File: Selenium/Activities/Activity18.java
package activity;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

public class Activity18 {

	public static void main(String[] args) {
		
		WebDriver driver = new ChromeDriver();
		
		driver.get("https://training-support.net/webelements/alerts");
		
		System.out.println("Title of the page: "+ driver.getTitle());
		
		driver.findElement(By.id("simple")).click();
		
		Alert alert = driver.switchTo().alert();
		System.out.println("Text of the Alert: " + alert.getText());
		
		alert.accept();
		
		driver.quit();
		
	}

}

File: Selenium/Activities/activity_18.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

with webdriver.Firefox() as driver:

    wait = WebDriverWait(driver, timeout=5)

    driver.get("https://training-support.net/webelements/alerts")

    print("Title of the page: ", driver.title)

    driver.find_element(By.ID, "simple").click()

    alert = wait.until(EC.alert_is_present())
    print("Text of the Alert: ", alert.text)

    alert.accept()
